packages/sdk-client/src/client.js

```diff
@@ -71,6 +73,11 @@ export default function createClient (options: ClientOptions): Client {
       fn: ProcessFn,
       opt: ProcessOptions = { accumulate: true },
     ): Promise<Array<Object>> {
+      if (!fn)
+        throw new Error('Missing argument! Second argument must be defined')
```

Comment:
🤔 How about validating only the request in a separate function, then validate `fn` directly here.

```js
// for "exec" method
validateRequest(request)

// for "process" method
validateRequest(request, { allowedMethods: ['GET'] })
if (!fn)
  throw new Error('The process function accepts a Function as a second argument that returns a Promise. See https://commercetools.github.io/nodejs/sdk/api/sdkClient.html#processrequest-processfn-options')

// function signature
function validateRequest (request, options) {
}
```